examples/string_token_substitution.ipynb

```diff
@@ -0,0 +1,247 @@
+{
+ "cells": [
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "# String token substitution",
+   "id": "4c4a5a5ed39eb2e1"
+  },
+  {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-06-05T09:33:38.946449Z",
+     "start_time": "2024-06-05T09:33:38.943885Z"
+    }
+   },
+   "cell_type": "code",
+   "source": "from resources import helpers",
+   "id": "8b19d0e7242a0a53",
+   "outputs": [],
+   "execution_count": 7
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "When a host application `resolve`s a string from the manager, it might be that the string contains placeholder tokens, which must be substituted with values held by the host application before the string can be used. Similarly, the host may publish an entity with a string property that contains placeholder tokens, which the manager must substitute before the string can be used.\n",
```

Comment:
[consider][minor]
```suggestion
    "When a host application `resolve`s a string from the manager, it is common that the string contains placeholder tokens, which must be substituted with values held by the host application before the string can be used. Similarly, the host may publish an entity with a string property that contains placeholder tokens, which the manager must substitute before the string can be used.\n",
```

Reply:
I don't think it's common. It's only really file paths that I've seen with such tokens. Other strings (scene graph paths, shader parameters, geometry up axis, ...) are less likely.